completions/puppet

```diff
@@ -289,6 +333,28 @@ _puppet()
                     return
             esac
             ;;
+        module)
+        case $prev in
```

Comment:
Indentation of module) block contents seems off, please follow style of existing blocks.